### DISCUSSION TITLE
Permission by tenant group

### DISCUSSION THREAD

**Discussion by jschewebbn** [netbox-community/netbox#8881] [Q&A] [answered]
I know how to create permissions that are limited by tenant, this works well. However I'm wondering if it's possible to create permission based on tenant group.

**Comment by drmsoffall** [comment, marked as answer]
You can use double underscores in constraints to access related objects/fields.  If you use `tenant` in your constraints now, you can use `tenant__group` to access the tenant's group.

**Reply by joyhu** [reply]
May I ask where exactly this is configured?

**Reply by bastianleicht** [reply]
This does not work for me, with the constraint "{"tenant__group": "test"}" i only get the Error:

> Invalid filter for <class 'django.contrib.auth.models.Group'>: Cannot resolve keyword 'tenant' into field. Choices are: id, name, object_permissions, permissions, user

**Reply by jeremystretch** [reply]
You need to specify the attribute of TenantGroup against which you're matching (presumably the name). The error message lists the available options. Try:

```
{"tenant__group__name": "test"}
```
